code/game/objects/items/weapons/tanks/tanks.dm

```diff
@@ -116,7 +116,7 @@
 	var/mob/living/carbon/human/H = user
 	user.visible_message("<span class='suicide'>[user] is putting [src]'s valve to [user.p_their()] lips! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	playsound(loc, 'sound/effects/spray.ogg', 10, 1, -3)
-	if (H && !QDELETED(H))
+	if (!QDELETED(H) && air_contents && air_contents.return_pressure() >= 1000)
```

Comment:
You need to base this off of the pressure differential. it shouldn't gib you if you do it in 500kpa overpressured room, but it should be easier in a vacuum.

In fact, if you wanted to quiet down some of the criticisms from players you could compromise and make it such that in a vacuum it does gib on stock e-tanks, but not in normal atmosphere.